.docker/entrypoint/lib/context.rb

```diff
@@ -15,11 +15,12 @@ class Context
       attr_reader :var_dir
       attr_reader :data_dir
       attr_reader :debug
+      attr_reader :profile
       attr_reader :auth_token
       attr_reader :configuration
       attr_reader :arguments
 
-      def initialize(env, var_dir, data_dir, auth_token = nil, debug = false)
+      def initialize(env, var_dir, data_dir, auth_token = nil, debug = false, profile = false)
```

Comment:
Codacy found an issue: [Line is too long. [94/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)